tests/Oryx.Integration.Tests/LocalDockerTests/PlatformEndToEndTestsBase.cs

```diff
@@ -0,0 +1,34 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.Oryx.Tests.Common;
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Oryx.Integration.Tests.LocalDockerTests
+{
+    public abstract class PlatformEndToEndTestsBase : IClassFixture<TestTempDirTestFixture>
+    {
+        private static readonly Random _rand = new Random();
+        protected readonly HttpClient _httpClient = new HttpClient();
+
+        protected static int GetRandomHostPort()
+        {
+            return _rand.Next(8000, 9000);
```

Comment:
Thinking more, does this guarantee uniqueness? I wonder if we will hit an issue down the road where multiple tests try to listen on the same port.
If we step back and think, do we really need this?...like these tests were stable (with exception of sql server tests which was a problem where the container took a long time to come up) with their statically assigned host ports. My vote would be to move back to that model unless we see a strong reason to not to.

Reply:
Moved the ports to a single file (Constants.cs), so it'll be easier to prevent double assignments in the future.